app/controllers/questionnaire_controller.rb

```diff
@@ -28,10 +28,8 @@ def create_informed_consent
 
   def create
     response_content = ResponseContent.create!(content: questionnaire_content)
-    @response.update_attributes!(content: response_content.id,
-                                 completed_at: Time.zone.now,
-                                 filled_out_by: @protocol_subscription.person,
-                                 filled_out_for: @protocol_subscription.filling_out_for)
+    @response.update_attributes!(content: response_content.id)
+    @response.complete!
```

Comment:
ergens wel jammer dat je nu 2 updates moet doen, al ziet dit er wel clean uit. Aan de andere kant hoort response_content misschien ook wel bij een `complete!`

Reply:
Ja idd daar had ik ook aan gedahct maar ik had het juist gesplitst gelaten. Voor als we in de toekomst een Response als "completed" willen markeren ookal issie leeg (van als een meting optioneel is bijv.), maar je wel wilt kunnen aangeven dat de gebruiker m heeft afgevinkt. Bijvoorbeeld een incidentmeting: er was geen incident dus ik vul geen vragenlijst in maar ik heb de vragenlijst ook niet gemist (ik wil niet mijn streak onderbreken of wat dan ook). Ik weet niet of we dat ooit willen gaan supporten maar daarom had ik het gesplitst gelaten.